readthedocs/oauth/migrations/0013_data_migration_for_remote_relation_model.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.2.16 on 2020-10-05 06:15
+
+import json
+from itertools import islice
+
+from django.db import migrations
+
+
+def move_data_to_remote_relations(apps, schema_editor):
+    RemoteRelation = apps.get_model('oauth', 'RemoteRelation')
+
+    def remote_relations_generator(relations, batch_size):
+        for relation in relations.iterator(chunk_size=batch_size):
+            relation.account_id = relation.remoterepository.account_id
+            relation.active = relation.remoterepository.active
+            relation.admin = relation.remoterepository.admin
+            relation.pub_date = relation.remoterepository.pub_date
+            try:
+                relation.json = json.loads(relation.remoterepository.json)
+            except json.decoder.JSONDecodeError:
+                pass
```

Comment:
It would be good to log something here with some data. At least to be warned that we couldn't migrate some data.